.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "@10up/eslint-config",
```

Comment:
@ryanwelcher Can you try using the new eslint config? https://github.com/10up/eslint-config/pull/17 It's still being worked on but you can already install it with `npm install @10up/eslint-config@next` there's a Gutenberg specific config

Any feedback is appreciated.